regression/storage.py

```diff
@@ -52,15 +53,16 @@ def setUp(self):
         self.case_buckets_to_delete = []
 
     def tearDown(self):
-        for bucket in self.case_buckets_to_delete:
-            bucket.delete()
+        with Batch(CONNECTION) as batch:
```

Comment:
Yeah being able to test on real data is useful.

We can also just check if GET requests go through in the same way, right?